static/js/plotjs/foldphase.js

```diff
@@ -18,6 +50,76 @@ for (let i = 0; i < n_labels; i++) {
     foldb.data.mjd_foldb[m] = folda.data.mjd_folda[m] + 1;
     foldberr.data.xs[m] = [foldb.data.mjd_foldb[m], foldb.data.mjd_foldb[m]];
   }
+
+  if (binsize > 1) {
+    var j;
```

Comment:
What if you push the whole sorting mechanism into the smoothing function and then you'll just call it for 'a' and 'b' separately?

Reply:
I think the main issue here is in the variable names

Reply:
What do you mean? you are doing the exact same logic (sorting then smoothing) on two arrays. 
Why not put each array into a function that returns the sorted+smoothed values?

Reply:
Ah got ya. Ok implemented.